packages/dynamic-links/RNFBDynamicLinks.podspec

```diff
@@ -25,7 +25,7 @@ Pod::Spec.new do |s|
   s.dependency          'React'
   s.dependency          'Firebase/Core', firebase_sdk_version
   s.dependency          'Firebase/DynamicLinks', firebase_sdk_version
-  s.dependency          'GoogleUtilities/AppDelegateSwizzler', '~> 6.2.3'
+  s.dependency          'GoogleUtilities/AppDelegateSwizzler', '~> 6.3.0'
```

Comment:
This was breaking XCode 11 build

Reply:
Can just remove this line entirely - have done so on master if you sync up